### PR TITLE
Level 1 complete

### PR DESCRIPTION
**Pull request by Zhongli5712**

### CODE REVIEW

**Comment by Zhongli5712** [comment]
Level 1 Complete

File: src/main/java/Duke.java [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+            execute(input);
+            input = scanner.nextLine(); //ready for next input
+        }
+
+        bye();
+    }
+
+    private static void begin() {
+        endCommand();
+
```
Comment (heeeyi) [inline src/main/java/Duke.java:20]: This repo contains a trivial amount of code, I will do the backup review instead

File: src/main/java/Duke.java [hunk 2]
```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        begin();
+
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
```
Comment (shenchenzizoe) [inline src/main/java/Duke.java:9]: jiayous

File: src/main/java/Duke.java [hunk 3]
```diff
@@ -1,10 +1,45 @@
...
+        while (!input.equals("bye")) {
+            execute(input);
+            input = scanner.nextLine(); //ready for next input
+        }
+
+        bye();
+    }
+
+    private static void begin() {
+        endCommand();
```
Comment (shenchenzizoe) [inline src/main/java/Duke.java:19]: I suppose this begin method has something before the endCommand()?

File: src/main/java/Duke.java [hunk 4]
```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        begin();
+
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+
+        while (!input.equals("bye")) {
```
Comment (shenchenzizoe) [inline src/main/java/Duke.java:10]: Proper use of while loop

**Review by shenchenzizoe — COMMENTED** [review]
a few basic structures are established

**Pull request closed without merging**